vendor/knative.dev/pkg/test/spoof/spoof.go

```diff
@@ -87,8 +87,7 @@ type SpoofingClient struct {
 	Client          *http.Client
 	RequestInterval time.Duration
 	RequestTimeout  time.Duration
-
-	logf logging.FormatLogger
```

Comment:
What's wrong with new?

Reply:
It requires parameters that are either hard to get in our test or we don't need, e.g. `kubeClient`, `resolvable` and `endpointOverride`.